access-mgmt/README.adoc

```diff
@@ -0,0 +1,469 @@
+* `Describe configs`
+
+|`Transactional ID`
+
+(For producer access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Write`
+* `Describe`
+
+|`Kafka instance`
+
+(For access to Kafka instance permissions in ACLs)
+|None
+a|
+* `Allow`
+* `Deny`
+a|
+* `Alter`
+* `Describe`
+|===
+
+[role="_additional-resources"]
+.Additional resources
+https://kafka.apache.org/documentation/#security_authz_primitives[Authorization Primitives^] in Kafka documentation
+
+[id="proc-setting-permissions_{context}"]
+== Setting account permissions in a Kafka instance in {product}
+
+[role="_abstract"]
+In {product}, you can create Access Control Lists (ACLs) in your Kafka instances and set permissions for how other user accounts or service accounts can interact with an instance and its resources. You can manage access for only the Kafka instances that you create or for the instances that the owner has enabled you to access and alter.
+
+.Prerequisites
+* You've created a Kafka instance and the instance is in *Ready* state.
+* The user account or service account that you're setting permissions for has been created in the organization.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to set permissions for.
+. Click the *Permissions* tab to view the current ACL permissions for this instance.
++
+--
+By default, new Kafka instances contain the following permissions. These permissions allow all accounts in the organization to view the instance permissions and to view topics in the instance, but not to produce or consume messages.
+
+NOTE: The wildcard character `*` denotes any occurrences of the specified resource. For example, `Topic` = `is *` means any topic in a Kafka instance.
+
+.Default ACL permissions for new Kafka instances
+[cols="25%,25%,25%,25%"]
+|===
+h|Account
+h|Resource
+h|Access type
+h|Operation
+
+|`All accounts`
+|`Topic is *`
+
+(Any topic)
+|`Allow`
+|`Describe`
+
+|`All accounts`
+|`Consumer group is *`
+
+(Any consumer group)
+|`Allow`
+|`Describe`
+
+|`All accounts`
+|`Kafka instance`
+
+(Kafka instance permissions in ACLs)
+|`Allow`
+|`Describe`
+|===
+--
+. Use the *Permissions* page to set permissions for a new account, add permissions to an existing account, or delete account permissions in this instance.
++
+To set permissions for a new account in this instance, follow these steps:
+
+.. Click *Manage access*.
+.. In the *Account* drop-down menu, select the new user account or service account that you want to set permissions for. You can also select *All accounts* to set permissions for all user accounts and service accounts in the organization.
+.. Click *Next*.
+.. Under *Review existing permissions*, verify the current permission settings for this instance. You can delete existing permissions now if needed, or you can wait to delete existing permissions later from the main *Permissions* page.
```

Comment:
page, or tab? Have seen it referred to both ways and want to be sure we're following our own style guide on UI item labels. :)

Reply:
Doh, I missed this in my earlier comments because I was searching for "tab".

We are changing the label *Permissions* to *Access*. 

We also refer to this as a _tab_ in the UI, but in this case I'm wondering if _main page_ makes more sense. When we say _Access tab_, the user's current context is somewhere else besides that tab. But if you are already in that tab, and open the modal, then how do you refer to the contents _behind_ the modal. I actually don't mind the use of _main page_ in this case to refer to the contents behind the modal.

Reply:
Right, this has come up a few times. The tab is what you click, open, etc., the page is what you view, go to, navigate to, etc. Just as in real life: You thumb to the tab to view a page. So as @jgiardino described.

So in sum, it's "Click the *Access* tab" but "In the *Access* page, ..." Same with *Connection* in the drawer: "Click the *Connection* tab, ...", but "In the *Connection* page, ..." etc.

Reply:
👍 Thanks for the clarification! This distinction makes sense for the Access tab/page. 
The Connection example seems a little odd to refer to it as a page, but it's not like I have a better suggestion. 🤷

Reply:
Stetson, if you have what you need here, I'm okay with resolving this - didn't want to do so in case you still needed this info.